pkg/driverbuilder/builder/build.go

```diff
@@ -15,3 +17,9 @@ type Build struct {
 	CustomBuilderImage string
 	KernelUrls         []string
 }
+
+func (b *Build) KernelReleaseFromBuildConfig() kernelrelease.KernelRelease {
+	kv := kernelrelease.FromString(b.KernelRelease)
```

Comment:
Hmm, just curious: why does `kernelrelease.FromString` not accept an object of type `*Build` and then try to populate as many fields as it can (including, in this case, `.Architecture`)?